dev/io.openliberty.microprofile.reactive.messaging30.internal_fat_tck/fat/src/io/openliberty/microprofile/reactive/messaging30/tck/ReactiveMessagingTCKLauncher.java

```diff
@@ -53,7 +54,11 @@ public static void tearDown() throws Exception {
 
     @Test
     @Mode(TestMode.FULL)
-    @AllowedFFDC({ "org.jboss.weld.exceptions.DeploymentException", "com.ibm.ws.container.service.state.StateChangeException", "jakarta.enterprise.inject.spi.DeploymentException" }) // The tested deployment exceptions cause FFDC so we have to allow for this.
+    @AllowedFFDC({ "javax.management.InstanceNotFoundException, org.jboss.weld.exceptions.DeploymentException", "com.ibm.ws.container.service.state.StateChangeException",
```

Comment:
This is broken, you have forgotten to put `"` in between the exceptions, so the tck fails due to the fact the deployment exceptions are now no longer allowed